database/factories/AchievementSetFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\AchievementSet;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<AchievementSet>
+ */
+class AchievementSetFactory extends Factory
+{
+    protected $model = AchievementSet::class;
+
+    public function definition(): array
+    {
+        return [
+            'user_id' => User::factory(),
```

Comment:
Not sure I understand why a set would have a user. A set is a group of achievements, which can each have their own unique author.

Also, this creates new users. Shouldn't it just pick a random developer from the database, or preferably, a user associate to one of the achievements in the set?

Reply:
> Not sure I understand why a set would have a user. A set is a group of achievements, which can each have their own unique author.

I agree. I didn't question why `user_id` was on the table, but I don't think a set should necessarily be ascribed to a user unless it's some kind of "community-made set" (??).

I've updated the seeder to respect that `user_id` is nullable, and it will no longer set a value for that field.